tests/tests/tier0/bluechictl-status-watch-service/python/bluechictl_status_watch.py

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright Contributors to the Eclipse BlueChi project
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import logging
+import re
+import unittest
+
+from bluechi_machine_lib.bluechictl import BackgroundRunner, SimpleEventEvaluator
+from dasbus.error import DBusError
+
+from bluechi.api import Node
+
+LOGGER = logging.getLogger(__name__)
+node_foo_name = "node-foo"
+simple_service = "simple.service"
+
+
+class TestBluechictlStatusWatch(unittest.TestCase):
+
+    def setUp(self) -> None:
+        service_status_active = re.compile(r"^" + simple_service + ".* active")
+        service_status_inactive = re.compile(r"^" + simple_service + ".* inactive")
+        expected_patterns = {service_status_active, service_status_inactive}
+        args = ["status", node_foo_name, simple_service, "-w"]
+        self.evaluator = SimpleEventEvaluator(expected_patterns)
+        self.bgrunner = BackgroundRunner(args, self.evaluator)
+
+    def test_bluechictl_status_watch(self):
+        self.bgrunner.start()
+
+        # stop the bluechi controller service to trigger a disconnected message in the agent
+        # hacky solution to cause a disconnect:
+        #   stop the controller service
+        # problem:
+        #   dasbus will raise a DBusError for connection timeout on v1.4, v1.7 doesn't
+        #   the command will be executed anyway, resulting in the desired shutdown of the node
+        #   therefore, the DBusError is silenced for now, but all other errors are still raised
+        try:
+            node = Node(node_foo_name)
+            node.stop_unit(simple_service, "replace")
+            node.start_unit(simple_service, "replace")
+            node.stop_unit(simple_service, "replace")
+        except DBusError:
+            pass
+        self.bgrunner.stop()
+        assert self.evaluator.processing_finished()
```

Comment:
The comment does not apply to this test so it should be removed. 
The same applies to the try-except. It is not needed - skipping any thrown error is actually bad here since it covers up a bug.  
```suggestion
        node = Node(node_foo_name)
        node.stop_unit(simple_service, "replace")
        node.start_unit(simple_service, "replace")
        node.stop_unit(simple_service, "replace")
        
        self.bgrunner.stop()
        assert self.evaluator.processing_finished()
```